javascript/ql/lib/change-notes/2025-04-07-websocket.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* Improved `WebSocket` analysis by refactoring the model to use API graphs.
```

Comment:
```suggestion
* Improved detection of `WebSocket` and `SockJS` usage.
```
I'd try to avoid the term "API graphs" as external users won't know what it means